experimental/auth-logic/wrappers/endorsement_wrapper_test.go

```diff
@@ -12,14 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package authlogic
+// Package wrappers contains an interface for writing wrappers that consume
```

Comment:
Again my understanding is that you don't need to repeat the package comment in every file, but should add file-specific comments after the `package` statement. Or does that not work with `golint`?

Reply:
OK I renamed the file `wrapper_interface.go` into `wrappers` (the same name as the package) so that we can more easily apply the rule Tiziano left in the other PR about putting the package comment just in the file named after the package. (I tried to find this link but couldn't).